examples/postgresql-ha/main.tf

```diff
@@ -83,6 +83,7 @@ module "pg" {
     {
       name                  = "0"
       zone                  = "us-central1-a"
+      availability_type     = "ZONAL"
```

Comment:
ZONAL seems to be the default anyway, lets set one to the other value and assert expected in the test.

Reply:
Set this to REGIONAL and asserted accordingly.